### PR TITLE
Fix autocorrect in PlacePicker

### PR DESCRIPTION
**Pull request by jvaclavik**
- enable autoFocus
- turn off autocorrect: It's annoying to write locations on iOS with autocorrect:

screenshot 2019-02-13 at 11 34 28

### CODE REVIEW

File: apps/core/scenes/search/PlacePicker/PlacePickerContent.js [hunk 1]
```diff
@@ -83,10 +83,11 @@ class PlacePickerContent extends React.Component<Props, State> {
         <View>
           <Text weight="bold">{this.getLabel()}:</Text>
           <TextInput
-            placeholder="write a place..."
+            placeholder="Write a place..."
             onChangeText={this.handleChangeText}
             value={this.state.text}
             prefix={<Icon name="search" />}
+            autoCorrect={false}
```
Comment (JoseDuSV) [inline apps/core/scenes/search/PlacePicker/PlacePickerContent.js]: It doesn't seem to work. Sadly we are using `universal-components` version of `TextInput`.
Where this props is not passed down. So we will need to update it first.

Comment (FilipMessa) [reply apps/core/scenes/search/PlacePicker/PlacePickerContent.js]: I will do that, I currently working on autoFocus for TextInput so this should be easy.

Comment (RobinCsl) [reply apps/core/scenes/search/PlacePicker/PlacePickerContent.js]: cf https://github.com/kiwicom/universal-components/pull/245

Comment (FilipMessa) [reply apps/core/scenes/search/PlacePicker/PlacePickerContent.js]: done

**Comment by tbergq** [comment]
Looks good, just a little improvement which I think could be done within this PR. It is standard behaviour that you dismiss the keyboard on scroll. Now you cannot select the bottom options, because they are covered by the keyboard. 

just add `onScroll={Keyboard.dismiss}` to the `FlatList`

**Comment by jvaclavik** [comment]
> Looks good, just a little improvement which I think could be done within this PR. It is standard behaviour that you dismiss the keyboard on scroll. Now you cannot select the bottom options, because they are covered by the keyboard.
> 
> just add `onScroll={Keyboard.dismiss}` to the `FlatList`


I was thinking about adding `keyboardShouldPersistTaps="always"` to FlatList. You could press the item without necessity of dismissing keyboard. Maybe we could combine it with your proposed change, what do you think? I tested both in the simulator and it works fine.

**Comment by tbergq** [comment]
> I was thinking about adding `keyboardShouldPersistTaps="always"`

Isn't the current behaviour that the modal closes when I tap a location?

**Comment by jvaclavik** [comment]
Right now it works like in the Kiwi.com app. When you scroll it closes the keyboard and when you don't scroll and click on the item, location is picked (you don't have to click once to close keyboard and then click to pick the location).

**Pull request merged**